### PR TITLE
Create category and product model

### PR DESCRIPTION
**Pull request by Deepak-Testpress**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ Django==4.1.3
 filelock==3.8.0
 identify==2.5.8
 nodeenv==1.7.0
+Pillow==9.3.0
```
Comment (waranlogesh) [inline requirements.txt:9]: This shouldn't be part of this PR. The title mentions you are creating a model for category and product. So add this here is side effect

**Review by waranlogesh — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (Deepak-Testpress) [reply requirements.txt:9]: I have added this Pillow to support images in models since we have dealt with images in our models and in order to make migrations I had to add this pillow before migrations! so it seemed like this belonged here..isn't it bro!
